notebooks/1_training.ipynb

```diff
@@ -164,7 +159,7 @@
     "data_dict = get_data_dict(nr_households, region)\n",
     "\n",
     "# Define Microgrid Env\n",
-    "env = MicrogridEnv(data_dict, wind=wind, wind_generator=wind_generator, alternative_cost=alternative_cost)\n",
```

Comment:
On my machine, the original code failed since these parameters were not defined in the `MicrogridEnv` class.